### DISCUSSION TITLE
关于 extra 包的小建议

### DISCUSSION THREAD

**Discussion by ghost** [v2fly/v2ray-core#1768] [Ideas]
今天注意到 v4.45.0 的二进制分发中不包含 extra 包，作为可选的扩充性质功能，该包在没有更新时不在新分发中重复出现也许是合理的，但是从（个人稍显自私的）实用角度，这对自己（其实就是我）写的笨笨的自动更新脚本不太友好。我是外行，想弱弱地建议一下，如果不会引起这里未虑及的其他潜在问题，希望让二进制分发一致地包含该包，顺便感恩开发者。

**Comment by AkinoKaede** [comment]
It's just an accident, the package had been re-uploaded.

**Reply by ghost** [reply]
Got it. Thanks for your reply!
